template-donation.php

```diff
@@ -178,6 +179,9 @@
 
 ?>
 
+<!-- Including a newer version of jQuery here needed for PayPal functionality -->
+<!-- We can remove the redundant jQuery include once we move away from WordPress -->
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Is there any reason we're holding on to the wp older version of jquery. Can we not just remove it from functions.php tji_scripts() and user the updated version you use here?

Reply:
Reading about that, it looks like it's highly recommended not to change jquery versions due to some core WordPress functions that apparently break. I don't know how serious those issues would be for us, but didn't to go down that path.

Since we plan to move away from WordPress (which will also help address the issue regarding the database), I thought we can do it this way for the time being.